modules/sbt-plugin/src/main/scala/ch/epfl/scala/debugadapter/sbtplugin/DebugAdapterPlugin.scala

```diff
@@ -62,7 +62,9 @@ object DebugAdapterPlugin extends sbt.AutoPlugin {
     val startRemoteDebugSession = inputKey[URI](
       "Start a debug session on a remote process"
     ).withRank(KeyRanks.DTask)
-
+    val debugConfig = settingKey[DebugConfig](
```

Comment:
We should try to avoid name conflicts in all sbt plugins. `debugConfig` seems to be a pretty common name, so we should make it more likely to be unique. I suggest `debugAdapterConfig`.

```suggestion
    val debugAdapterConfig = settingKey[DebugConfig](
```